scripts/01_IAvH_test/protcon_function.yml

```diff
@@ -0,0 +1,28 @@
+script: protcon_function.R
+description: "This sample script shows how it works."
+external_link: https://github.com/GEO-BON/biab-2.0
+inputs:
+  data_areas:
+    label: data_areas
+    description: data_areas
+    type: text
+    example: C:/Users/LENOVO/Desktop/pipe_shiny/output/load_collection/load_collection.R/test/output.json

```

Comment:
If you want it to be usable directly in the runner, you can use the path of the output folder inside the runner. It would look like this:
/output/load_collection/load_collection.R/test/output.json

The runner itself is a linux docker.
(Same for other windows C: paths)